conans/client/tools/files.py

```diff
@@ -272,3 +272,13 @@ def _get_possible_filenames(filename):
                         return trick_path
 
     return None
+
+
+def unix2dos(filepath):
+    """
+    Converts line breaks in a text file from Unix format (LF) to DOS format (CRLF)
+    :param filepath: Path to the text file to convert
+    :return:
+    """
+    contents = open(filepath, 'r').read()
+    open(filepath, 'w').write(contents.replace('\n', '\r\n'))
```

Comment:
Have you check the ``normalize()`` function in conans/util/files.py?

The thing is that you have to be careful that the file doesn't already have \r\n, otherwise you would be adding more LF than desired. The regex used in normalize() is:

```python
def normalize(text):
    if platform.system() == "Windows":
        return re.sub("\r?\n", "\r\n", text)
    else:
        return text
```
It also makes sense to condition it to Windows only, I guess you don't want to do that in other OS, do you?
Would you have a use case in which you would be using this helper?

Thanks for contributing!